### PR TITLE
Add Aquatis to host list

### PR DESCRIPTION
**Pull request by AmiT177**

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -2208,5 +2208,15 @@
     "plan": "",
     "reviewed": "2019.7.22",
     "note": ""
-  }
+  },
+  {
```
Comment (ohemorange) [inline _data/hosting_providers.json]: Order matters; you'll want to add this entry alphabetized in the right section.

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -2208,5 +2208,15 @@
...
+    "name": "Aquatis",
+    "link": "https://aquatis.host/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "2019.8.31",
+    "note": ""
```
Comment (ohemorange) [inline _data/hosting_providers.json]: Do you provide certificates for all of your hosting products? If not, we may want to add a note saying which products are supported.

File: _data/hosting_providers.json [hunk 3]
```diff
@@ -2208,5 +2208,15 @@
...
     "reviewed": "2019.7.22",
     "note": ""
-  }
+  },
+  {
+    "name": "Aquatis",
+    "link": "https://aquatis.host/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
```
Comment (ohemorange) [inline _data/hosting_providers.json]: Is there a page on your site that says that you support TLS by default? We should link to that.

File: _data/hosting_providers.json (hunk 2 above)
Comment (AmiT177) [reply _data/hosting_providers.json]: Yeah, all of the hosting products have https included.

File: _data/hosting_providers.json (hunk 3 above)
Comment (AmiT177) [reply _data/hosting_providers.json]: On the product page it shows up, didn't know if I should link it or not.
https://you.aquatis.host/cart.php?gid=2

Comment (ohemorange) [reply _data/hosting_providers.json]: hm, you're right, no need. just to check -- there's no steps users need to take or buttons to click to get https support?

Comment (AmiT177) [reply _data/hosting_providers.json]: Yap, they should get https automatically. (we have AutoSSL with cPanel)

File: _data/hosting_providers.json (hunk 1 above)
Comment (ohemorange) [reply _data/hosting_providers.json]: this note was resolved but I'm still seeing it at the end of the list; is there a commit that needs to be pushed?

**Pull request merged**